src/mn_menu.c

```diff
@@ -693,6 +695,24 @@ static void M_VerifyNightmare(int ch)
 
 static void M_ChooseSkill(int choice)
 {
+    if (!M_ParmExists("-coop_spawns"))
+    {
+        coop_spawns = false;
+    }
+
+    if (W_CheckNumForName("M_ANME") >= 0)
```

Comment:
If you recall the master levels `M_DOOM` issue, the concern there was wad or lump specific hacks. I know ports are full of them already, but maybe let's not add new ones when possible (what if another wad has a lump named `M_ANME`?). I think the suggestions of either a custom skill menu or some kind of standards update seems like a better way to go, but more work of course.

Reply:
We need to add checksum for WADs to avoid lump name collision. It will also be useful for compatibility database, not all popular levels are playbale on default MBF21 compatibility, see: https://github.com/kraflab/dsda-doom/blob/master/prboom2/src/dsda/compatibility.c
Another use for it will be to check wads for multiplayer (Choco has it, I removed it).

Eviternity 2 is popular, and this new skill is actually a balanced and interesting addition.

Reply:
That compatibility database will have to be updated whenever a new version of a wad is released, e.g. RC6 to final for Eviternity II, which means issues will return until yet another Woof release that adds the new checksum to a database entry. That approach works for old wads, but even then it's not guaranteed as authors like to do updates years later.

Reply:
Yes, it is problematic for Eviternity 2 case. Then I vote to wait until new standard, for now we can tell users to use `-coop_spawn` parameter. Coop spawns is not interesting in most WADs, not sure it is worth to expose it in the menu.